### PR TITLE
Move ARM swagger to subfolder

### PR DESCRIPTION
**Pull request by petrkotas**
### Which issue this PR addresses:

The PR prepares the ARORP ARM API for the inclusion of HCP RP in the ARM manifest folder.
It moves the files to `openshiftclusters` subfolder and updates automatic generation accordingly.

Fixes

### CODE REVIEW

**Review by mociarain — DISMISSED** [review]
I'm not sure I understand the full implications of this but if it's just re-structuring things then it LGTM

**Review by anshulvermapatel — DISMISSED** [review]
LGTM

File: pkg/api/v20220904/generate.go [hunk 1]
```diff
@@ -3,4 +3,4 @@ package v20220904
 // Copyright (c) Microsoft Corporation.
 // Licensed under the Apache License 2.0.
 
-//go:generate go run ../../../hack/swagger github.com/Azure/ARO-RP/pkg/api/v20220904 ../../../swagger/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/stable/2022-09-04
+//go:generate go run ../../../hack/swagger github.com/Azure/ARO-RP/pkg/api/v20220904 ../../../swagger/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift//openshiftclusters/stable/2022-09-04
```
Comment (kimorris27) [inline pkg/api/v20220904/generate.go:6]: There is an extra slash before `openshiftclusters` that we could probably remove to be safe even though it didn't seem to cause problems yet.

**Review by kimorris27 — CHANGES_REQUESTED** [review]

File: pkg/api/v20220904/generate.go (hunk 1 above)
Comment (petrkotas) [reply pkg/api/v20220904/generate.go]: done, good catch

**Review by kimorris27 — DISMISSED** [review]

**Comment by tiguelu** [comment]
Closing. This PR has been moved to #3805

**Pull request closed without merging**